templates/rosettanet-decode-response.json

```diff
@@ -0,0 +1,167 @@
+                    "trackingId": "@outputs('RosettaNet_Decode')?['body']?['trackingId']"
+                  }
+                }
+              },
+              "runAfter": {
+                "RosettaNet_Decode": [
+                  "Succeeded"
+                ]
+              },
+              "expression": {
+                "and": [
+                  {
+                    "not": {
+                      "equals": [
+                        "@outputs('RosettaNet_Decode')?['body']?['outboundSignal']",
+                        null
+                      ]
+                    }
+                  }
+                ]
+              },
+              "type": "If"
+            },
+            "RosettaNet_Decode": {
+              "runAfter": {},
+              "type": "RosettaNetDecode",
+              "inputs": {
+                "homeRole": "Initiator",
+                "messageHeaders": {
+                  "x-RN-Response-Type": "Async",
+                  "x-RN-Version": "RosettaNet/V02.00"
+                },
+                "messageToDecode": "@triggerBody()"
+              }
+            },
+            "Send_decoded_message_to_LOB_application": {
+              "runAfter": {
+                "RosettaNet_Decode": [
+                  "Succeeded"
+                ]
+              },
+              "type": "ApiConnection",
+              "inputs": {
+                "body": "@outputs('RosettaNet_Decode')?['body']?['messageContent']",
+                "host": {
+                  "connection": {
+                    "name": "@parameters('$connections')['azureblob']['connectionId']"
+                  }
+                },
+                "method": "post",
+                "path": "/datasets/default/files",
+                "queries": {
+                  "folderPath": "/public",
+                  "name": "RNMessages",
+                  "queryParametersSingleEncoded": true
+                }
+              },
+              "runtimeConfiguration": {
+                "contentTransfer": {
+                  "transferMode": "Chunked"
+                }
+              }
+            }
+          },
+          "outputs": {}
+        },
+        "connectionReferences": {
+            "azureblob": {
+			"connection": {
+				"id": ""
+			},
+			"api": {
+				"id": "/subscriptions/{0}/providers/Microsoft.Web/locations/{1}/managedApis/azureblob"
+			}
+        }
+        },
+        "apiSummaries": [
+           {
+                "type": "request",
+                "displayName": "${Resources.REQUEST_TRIGGER_DISPLAYNAME}",
+                "iconUri": "${Images.RequestIcon}",
+                "brandColor": "#009DA5"
+            },
+            {
+                "type": "RosettaNetDecode",
+                "displayName": "RosettaNet Decode",
+                "iconUri": "https://latemplateartifacts.blob.core.windows.net/rosettanet/rosetta-net-connector-100.png",
```

Comment:
Who owns this blob?

Reply:
I dont know, I just copied it from an existing template. I can upload the icon in my storage if thats preferable